tests/ResponseTest.php

```diff
@@ -111,6 +113,71 @@ public function testSetContent()
         $this->assertSame('Hello World', (string) $response->getBody());
     }
 
+    public function testSetContentWithResource()
+    {
+        $response = new Response();
+
+        $this->assertSame($response, $response->setContent(fopen('php://input', 'rb')));
+        $this->assertInstanceOf(MessageBody::class, $response->getBody());
+        $this->assertInstanceOf(MessageBodyResource::class, $response->getBody());
+        $this->assertSame(0, $response->getBody()->getSize());
+    }
+
+    public function testIsStatusCode()
+    {
+        $response = new Response();
+
+        $this->assertTrue($response->isStatusCode(200));
+        $this->assertFalse($response->isStatusCode(400));
+    }
+
+    /**
+     * @expectedException        \RuntimeException
+     * @expectedExceptionMessage Could not parse response (error 1).
+     */
+    public function testParseShouldThrowRuntimeExceptionError1()
+    {
+        Response::parse('<!DCOTYPE html><html>HTML strings</html>');
```

Comment:
Is the typo in DOCTYPE on purpose? It's not really useful as it's already an invalid response anyway. I would fix it.

Reply:
I think the ```DOCTYPE``` is needed because it can declare the HTML version.
Please see this [link](https://www.w3schools.com/tags/tag_doctype.asp).

Reply:
No, I meant, you wrote `DCOTYPE` instead of `DOCTYPE`!

Reply:
Oh right! I fix the typo.....